# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Shape.h
# Commit: 40d2e0691861
# Full Hash: 40d2e069186171b8b1389e3388fad5927accbd7e
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 10 - Stop passing proto in a few places now that we can get it from the shape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106980
# ==============================================================================

diff -r ec72f104edbf -r 40d2e0691861 js/src/vm/Shape.h
--- a/js/src/vm/Shape.h	Sat Mar 06 01:05:22 2021 +0000
+++ b/js/src/vm/Shape.h	Sat Mar 06 01:05:23 2021 +0000
@@ -1406,8 +1406,7 @@
    * getInitialShape calls, until the new shape becomes unreachable in a GC
    * and the table entry is purged.
    */
-  static void insertInitialShape(JSContext* cx, HandleShape shape,
-                                 HandleObject proto);
+  static void insertInitialShape(JSContext* cx, HandleShape shape);
 
   /*
    * Some object subclasses are allocated with a built-in set of properties.
